cmd/cluster/core/dump.go

```diff
@@ -120,16 +123,31 @@ func dumpGuestCluster(ctx context.Context, opts *DumpOptions) error {
 		return fmt.Errorf("failed to create tempfile for kubeconfig: %w", err)
 	}
 	defer func() {
-		if err := kubeconfigFile.Close(); err != nil {
-			opts.Log.Error(err, "Failed to close kubeconfig file")
-		}
```

Comment:
why move this?

Reply:
Because we should have closed it before invoking the `oc` command to use it. It's a bug that we're waiting for the end of the function to close it.